gadgetron/meta.yaml

```diff
@@ -1,53 +1,53 @@
 package:
   name: gadgetron
-  version: 4.1.5
+  version: 4.2.1
 
 source:
-  git_rev: dff06e87388c2930359efebabf98c804cb0f00e3
+  git_rev: fcea9bd9b9f10781302a026d0fe3e4a0daf053ac
   git_url: https://github.com/gadgetron/gadgetron
 
 requirements:
   build:
     - armadillo>=9.900.5
     - boost=1.76.0
     - cmake>=3.20.0
-    - cuda-libraries-dev      # [linux64]
-    - cuda-libraries          # [linux64]
-    - cuda-runtime            # [linux64]
-    - cuda-nvcc               # [linux64]
+    - cuda-libraries-dev=11.6.1      # [linux64]
```

Comment:
I think we need cpr and howardhinnant_date

Reply:
Good call.